docs/index.md

```diff
@@ -3,19 +3,20 @@
   title: ""
 ---
 
-# AddressBook Level-3
-
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
+# Key Contacts
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2425S1-CS2103T-T08-2/tp/actions)
```

Comment:
should this be updated to our repo?

Reply:
the first link is for the image, second link is the one that links to our repo